cfme/services/myservice/ui.py

```diff
@@ -52,6 +52,7 @@ class toolbar(View):  # noqa
     policy_btn = Dropdown('Policy')
     lifecycle_btn = Dropdown('Lifecycle')
     download_choice = Dropdown('Download')
+    view_selector = View.nested(ItemsToolBarViewSelector)
```

Comment:
**Required**

This, and the widgets above to Configuration, are actually duplicate widgets for what's nested in toolbar at L38. I think you just need to update any call chains accessing these to instead use `view.toolbar.widget`

`reload` widget isn't in the toolbar above, and should be.

Reply:
Ah, the toolbar sub-class was probably introduced in a rebase from master, and I didn't see it.